docs/Overrides/NginxProxyManager.md

```diff
@@ -0,0 +1,98 @@
+# Nginx Proxy Manager
+[Nginx Proxy Manager](https://[https://nginxproxymanager.com/](https://nginxproxymanager.com/))  is  Docker container for managing Nginx proxy hosts and SSL Certificates with a simple, powerful interface.
+ 
+# Features of Nginx Proxy Manager
+ 
+### Get Connected
+Expose web services on your network · Free SSL with Let's Encrypt · Designed with security in mind · Perfect for home networks
+
+### Proxy Hosts
+Expose your private network Web services and get connected anywhere.
+
+### Beautiful UI
+Based on Tabler, the interface is a pleasure to use. Configuring a server has never been so fun.
+
+### Free SSL
+Built in Let’s Encrypt support allows you to secure your Web services at no cost to you. The certificates even renew themselves!
+
+### Multiple Users
+Configure other users to either view or manage their own hosts. Full access permissions are available.
+
+The GIT Repository for Nginx Proxy Manager is located at [[https://github.com/jc21/nginx-proxy-manager](https://github.com/jc21/nginx-proxy-manager)]
+### Config File
+Nginx Proxy Manager requires a Configuration file named `config.json`. This file needs to be in the appdata folder for NPM before it is started.
+
+```json
+{
+  "database": {
+    "engine": "mysql",
+    "host": "db",
+    "name": "npm",
+    "user": "npm",
+    "password": "npm",
+    "port": "3306"
+  }
+}
+```
+## Example Docker Compose Override
+
+```yaml
+#NGINX Proxy Manager with LetsEncrypt https://github.com/jc21/nginx-proxy-manager
+    version: "3.4"  # this must match the version in docker-compose.yml
+    services:
+        proxymanager:
+        image: jc21/nginx-proxy-manager:latest
+        container_name: proxymanager
+        labels:
```

Comment:
The labels aren't really necessary (if anything it's just extra and might confuse users). They don't do any harm, either.

Reply:
I was just copying your files. I can delete or continue to include them. Up to you.

Reply:
Let's take them out. I feel like this section (overrides) can/will grow and I don't want to continue the trend which might imply that labels are required.

Reply:
taking it out.